sympy/physics/continuum_mechanics/beam.py

```diff
@@ -1620,7 +1620,8 @@ def _solve_for_ild_equations(self):
         copy of the load equation and uses it to calculate shear force and bending
         moment equations.
         """
-
+        if self._applied_hinges:
+            raise NotImplementedError("I.L.D. calculations are not implemented for beams with hinges.")
```

Comment:
As this might be a next step, could you add this as an issue?

Reply:
I added two issues that have to do with this problem. 
Missing tests: #26602
Unable to handle hinge beams: #26601